internal/cmd/deploy/deploy.go

```diff
@@ -16,155 +16,48 @@
 package deploy
 
 import (
-	"context"
 	"fmt"
-	"time"
 
-	"github.com/mia-platform/miactl/internal/client"
 	"github.com/mia-platform/miactl/internal/clioptions"
-	"github.com/mia-platform/miactl/internal/resources"
 	"github.com/spf13/cobra"
 )
 
-const (
-	deployProjectEndpointTemplate  = "/api/deploy/projects/%s/trigger/pipeline/"
-	pipelineStatusEndpointTemplate = "/api/deploy/projects/%s/pipelines/%d/status/"
-)
-
 func NewDeployCmd(options *clioptions.CLIOptions) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "deploy ENVIRONMENT",
 		Short: "Deploy the target environment.",
-		Long: `Trigger the deploy of the target environment in the selected project.
+		Long: `Deprecation Warning: This command is deprecated. Use 'deploy trigger' instead.
+
+Trigger the deploy of the target environment in the selected project.
 
 The deploy will be performed by the pipeline setup in project, the command will then keep
 listening on updates of the status for keep the user informed on the updates. The command
 will exit with error if the pipeline will not end with a success.`,
 		Args: cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Print("Deprecation Warning: The 'deploy' command is deprecated. Use 'deploy trigger' instead.\n\n")
+
```

Comment:
Can't we use the `Deprecated` property of cobra command instead of this custom integration?